Builds/CMake/CMakeFuncs.cmake

```diff
@@ -610,11 +624,11 @@ macro(setup_build_boilerplate)
       /nologo              # Suppress login banner
       /W3                  # Warning level 3
       /WX-                 # Disable warnings as errors
-      /wd"4018"
-      /wd"4244"
-      /wd"4267"
-      /wd"4800"            # Disable C4800(int to bool performance)
-      /wd"4503"            # Decorated name length exceeded, name was truncated
+      /wd4018
+      /wd4244
+      /wd4267
```

Comment:
These look like they were copied over from `SConstruct`, and they aren't documented there either, so I'll look into this, though I may need to make the change later.